packages/common/src/util/http.js

```diff
@@ -77,31 +78,57 @@ const assertOK = async (response, errorMap, fullUrl, method, startTime) => {
   }
 };
 
+export const ERROR_ABSOLUTE_URL = 'Absolute URLs not suppored';
+
+// throws if a path is absolute
+export const assertRelativeUrl = path => {
+  if (/https?:\/\//.test(path)) {
```

Comment:
what if  it's `http`?, I need to test this

Reply:
Hey great question. The regex means the `https` is optional, but let me push a unit test to convince you

Reply:
Ok check 975cd9b4a1fc765b8c0bda80367e8b9aba3d336c for tests